src/modules/75-cf/pages/onboarding/OnboardingDetailPage.tsx

```diff
@@ -7,227 +7,236 @@
 
 import React, { useMemo, useState } from 'react'
 import { useHistory, useParams } from 'react-router-dom'
-import { Button, ButtonVariation, Container, FlexExpander, Layout, MultiStepProgressIndicator } from '@harness/uicore'
-import { Intent, Color } from '@harness/design-system'
+import {
+  Button,
+  ButtonVariation,
+  Container,
+  FlexExpander,
+  Heading,
+  Layout,
+  MultiStepProgressIndicator
+} from '@harness/uicore'
+import { Intent, Color, FontVariation } from '@harness/design-system'
+import { Divider } from '@blueprintjs/core'
 import { StepStatus } from '@common/constants/StepStatusTypes'
 import type { ApiKey, Feature } from 'services/cf'
 import routes from '@common/RouteDefinitions'
-import { useStrings } from 'framework/strings'
+import { StringKeys, useStrings } from 'framework/strings'
 import type { PlatformEntry } from '@cf/components/LanguageSelection/LanguageSelection'
 import { useTelemetry } from '@common/hooks/useTelemetry'
 import { Category, FeatureActions } from '@common/constants/TrackingConstants'
 import { CreateAFlagView } from './views/CreateAFlagView'
 import { SetUpYourApplicationView } from './views/SetUpYourApplicationView'
+import { SetUpYourCodeView } from './views/SetUpYourCodeView'
 import { ValidateYourFlagView } from './views/ValidatingYourFlagView'
 import css from './OnboardingDetailPage.module.scss'
 
-enum TabId {
-  CREATE_A_FLAG = 'created-a-flag',
-  SET_UP_APP = 'set-up-your-app',
-  TEST_YOUR_FLAG = 'test-your-flag'
-}
-
-enum STEP {
-  NEXT = 'next',
-  BACK = 'back'
-}
-
-enum STATUS {
+enum STEPS {
```

Comment:
```suggestion
enum STEP {
```

Don't think this should be a plural, as when you are referencing it you are referencing one step?